### PR TITLE
Added io.github.hakandundar34coding.system-monitoring-center into exceptions.json file

### PR DESCRIPTION
**Pull request by hakandundar34coding**

### CODE REVIEW

File: flatpak_builder_lint/staticfiles/exceptions.json [hunk 1]
```diff
@@ -1456,4 +1456,7 @@
     "org.srb2.SRB2Kart": {
         "flathub-json-modified-publish-delay": "extra-data"
     }
+    "io.github.hakandundar34coding.system-monitoring-center": {
+        "finish-args-flatpak-spawn-access": "the app predates this linter rule"
```
Comment (barthalion) [inline flatpak_builder_lint/staticfiles/exceptions.json:1460]: I mean, that's not true? Can you put actual use-case here?

Comment (hakandundar34coding) [reply flatpak_builder_lint/staticfiles/exceptions.json]: Updated the file again. A lot of applications used the previous explanation.

Comment (hfiguiere) [reply flatpak_builder_lint/staticfiles/exceptions.json]: because we had no explanation and we added the linter rules. This app does't predate the rule.

**Comment by razzeee** [comment]
This was missing a leading `,` and this our json is not a valid json anymore :/

**Comment by barthalion** [comment]
I'm aware, and fixing it right now.

**Comment by hfiguiere** [comment]
Fixed with #73 I just merged. I also opened #74 to safeguard from this.

**Pull request merged**